test/file_io.jl

```diff
@@ -18,8 +18,12 @@ end
         @test jlso["data"] == v
     end
 
-    @testset "unsupported julia version" begin
-        jlso = JLSOFile(v"1.0", VERSION, :julia_serialize, :none, img, pkgs, Dict("data" => hw_5))
+    @testset "unsupported julia version $_version" for _version in (v"1.0", v"2.0")
```

Comment:
really minor, but I was confused by this for a second, til i realised `_version` is `jlso_version` - perhaps rename?

Reply:
Can we do this?